moped-database/metadata/tables.yaml

```diff
@@ -1358,79 +1358,63 @@
           table:
             name: moped_subphases
             schema: public
-  insert_permissions:
+  select_permissions:
     - role: moped-admin
       permission:
-        check: {}
         columns:
-          - phase_average_length
-          - phase_description
-          - phase_id
-          - phase_name
-          - phase_order
           - required_phase
-    - role: moped-editor
-      permission:
-        check: {}
-        columns:
           - phase_average_length
-          - phase_description
           - phase_id
-          - phase_name
           - phase_order
-          - required_phase
-  select_permissions:
-    - role: moped-admin
-      permission:
-        columns:
-          - phase_average_length
           - phase_description
-          - phase_id
           - phase_name
-          - phase_order
-          - required_phase
+          - phase_key
```

Comment:
The only change here is the addition of select permissions on `phase_key` for all roles.

Reply:
I'm sorry about this diff 😩